e2e/test/prerequisites/readme.md

```diff
@@ -1,11 +1,34 @@
+
 # Azure IoT C# End-to-end test prerequisites
 
 The E2E tests require Azure services set up and configured with test devices (IoT Hub, Provisioning).
-Please copy the iot_config.* scripts to a secure location, remove the `_template` suffix and follow the comments to fill in the missing information.
 
-To generate required test certificates you can use the tools available [here](https://github.com/Azure/azure-iot-sdk-c/blob/master/tools/CACertificates/CACertificateOverview.md).
+- Navigate to reporoot\azure-iot-sdk-csharp\e2e\test\prerequisites\E2ETestsSetup
+
+- Open powershell in Administrator mode and run the following command by replacing the variables in brackets with your own preferred values.
+  
+  ```Shell
+  .\e2eTestsSetup -SubscriptionId [yourSubscriptionId] -Region [YourRegion] -ResourceGroup [ResourceGroupName] -GroupCertificatePassword [Password] -InstallDependencies [true/false]
+  ```
```

Comment:
Couple of thoughts:

Reply:
I think we can state something like:
use this script to create all resources that you need to run our entire E2E test suite, or go to this link (link to our resources generation script) to see which resources are required and what env variables they are associated with.

Reply:
1) In common errors, it is called out that the script is idempotent meaning, it will not create the resource again and again with a different name unless the user provides a different name. 
2) Given we want uses to run all the E2E tests before submitting any PRs, we are not creating anything unnecessary other than a KeyVault, which is very very negligible cost given how many transactions they will do with it. I don't think we really need to maintain a separate file. Those files are already out of sync.
3) It would fine to mention to look at the script but don't you think that is understood as the instructions say they create all the necessary resources? If they really want to selectively create resources, they are free to do so.

Reply:
Overall I am seeing this as a convenience we are providing to the user. But if they are an advanced user, they will have the expertise to selectively create what they want and run. I don't see too much value in giving them instructions on running every test. It would be easier if we wrote the tests in a way where each test class has a list of environment variables it needs (similar to samples) so that it is self explanatory. However our tests are not designed like that either so some env variables are just manadatory.

Reply:
I think this will be easier to discuss over a call :)